mcs/class/System.Net.Http/HttpClient.DefaultHandler.cs

```diff
@@ -0,0 +1,10 @@
+namespace System.Net.Http
```

Comment:
Not sure default is the right word, each profile will have a different default. This one is I guess WebRequestHandler, right?

Reply:
There is exactly one "default" that is used by the parameterless ("default") constructor - or in other words, the default constructor needs to call "something".  This PR allows this "something" to be profile-specific.  Does this answer your question?